jupyter-docker/Dockerfile

```diff
@@ -45,7 +45,9 @@ RUN echo "deb $DEBIAN_REPO/debian jessie main"                   > /etc/apt/sour
  && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
  && apt-get update \
  && apt-get install -yq --no-install-recommends \
-    google-cloud-sdk
+    google-cloud-sdk \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
I wonder if it makes sense to combine `apt-get clean && rm -rf /var/lib/apt/lists/*` into an alias like so:

```alias apt-get-cleanup='apt-get clean && rm -rf /var/lib/apt/lists/*'```